src/synopsys-action/tools-parameter.ts

```diff
@@ -60,6 +60,27 @@ export class SynopsysToolsParameter {
       }
     }
 
+    if (parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED)) {
+      info('Polaris PR comment is enabled')
+      const prCommentSeverities: string[] = []
+      const inputPrCommentSeverities = inputs.POLARIS_PRCOMMENT_SEVERITIES
+      if (inputPrCommentSeverities != null && inputPrCommentSeverities.length > 0) {
```

Comment:
Can't be just to _if (inputPrCommentSeverities)_ from this  
_if (inputPrCommentSeverities != null && inputPrCommentSeverities.length > 0) _

Reply:
want to be explicit and ensure that inputs.POLARIS_PRCOMMENT_SEVERITIES is both defined and contains a non-empty string

Reply:
we can use the truthy value, FYI ref : https://stackoverflow.com/a/5515349

Reply:
what if it contains only whitespaces

Reply:
If its working then okay,